nomad/rpc_test.go

```diff
@@ -1186,7 +1186,7 @@ func TestRPC_TLS_Enforcement_RPC(t *testing.T) {
 			name:   "local server/clients only rpc",
 			cn:     "server.global.nomad",
 			rpcs:   localClientsOnlyRPCs,
-			canRPC: false,
+			canRPC: true,
```

Comment:
This caused problem 1 to go unnoticed. Server certificates should be able to call client RPCs